src/app.ts

```diff
@@ -537,8 +537,14 @@ function registerPublicRoutes(app: Express): void {
   const packageJson = require("../package.json");
   const version = t.string.decode(packageJson.version).getOrElse("UNKNOWN");
 
+  // The minimum app version that support this API
+  const minAppVersion = t.string
+    .decode(packageJson.minAppVersion)
+    .getOrElse("0.0.0");
```

Comment:
this is a lie! 😂 it's probably better to reply with `UNKNOWN` and let the client decide what to do

Reply:
fixed in 2909928